build/e2e.mk

```diff
@@ -15,7 +15,14 @@ test-e2e-deps: ginkgo ## Install e2e test dependencies
 test-e2e-setup: ## Setup cluster for e2e tests (if not already setup)
 	@if ! kubectl get namespace mcp-system >/dev/null 2>&1; then \
 		echo "Setting up cluster for e2e tests..."; \
-		$(MAKE) local-env-setup; \
+		$(MAKE) tools; \
+		$(MAKE) kind-create-cluster; \
+		$(MAKE) build-and-load-image; \
+		$(MAKE) gateway-api-install; \
+		$(MAKE) istio-install; \
+		$(MAKE) metallb-install; \
+		$(MAKE) deploy-namespaces; \
+		$(MAKE) deploy-gateway; \
```

Comment:
this change is to slim down the test environment to just what's needed.
More could maybe be removed , but for now I removed keycloak and kuadrant.